hawkular-alerts-metrics/src/test/java/org/hawkular/alerts/external/metrics/ExpressionTest.java

```diff
@@ -37,8 +37,8 @@ public void expressionParseTest() {
         assertTrue(x.getThreshold() == 100.5);
         assertEquals(x.getMetric(), "foo");
 
-        x = new Expression("TAG:5m:avgd(bar <= 100),10");
-        assertEquals(x.getTarget(), Target.Tag);
+        x = new Expression("metric:5m:avgd(bar <= 100),10");
```

Comment:
Same comment, I guess if how to distinguish if bar is counter/gauge.
I guess by default is read as gauge.